File: 3685-count-subarrays-of-length-three-with-a-condition/3685-count-subarrays-of-length-three-with-a-condition.py
class Solution:
    def countSubarrays (self, nums : List [int]) -> int:
        count = 0
        for i in range (len (nums) - 2):
            if 2 * (nums [i] + nums [i + 2]) == nums [i + 1]:
                count += 1
        return count

File: 3685-count-subarrays-of-length-three-with-a-condition/3685-count-subarrays-of-length-three-with-a-condition.c
int countSubarrays (int* nums, int numsSize) {
    int ans = 0;
    for (int i = 0; i < numsSize; i++) {
        if (i <= numsSize - 3)
            if (nums [i] + nums [i + 2] == ((double) nums [i + 1]) / 2)
                    ans++;
    }
    return ans;
}